src/dune_rules/cinaps.ml

```diff
@@ -38,6 +39,7 @@ let decode =
          (Dune_lang.Syntax.since syntax (1, 1) >>> repeat Dep_conf.decode)
      and+ cinaps_version = Dune_lang.Syntax.get_exn syntax
      and+ alias = field_o "alias" Alias.Name.decode
+     and+ link_flags = Link_flags.Spec.decode ~since:(Some (3, 8))
```

Comment:
Note that the cinaps has its own syntax used for versioning. In this case, you should introduce `(1, 3)` to  fix the issue.